frontend/src/utils/useBiolink.js

```diff
@@ -43,24 +53,38 @@ export default function useBiolink() {
       obj[strings.nodeFromBiolink(item)] = hierarchy;
       return obj;
     }, {});
-    setHierarchies(newHierarchies);
+    return newHierarchies;
   }
 
   /**
    * Filter out concepts that are not derived classes of base class
    * @returns {array} list of valid concepts
    */
-  function getValidConcepts() {
-    const newConcepts = Object.keys(hierarchies).filter((key) => hierarchies[key].includes(baseClass));
-    setConcepts(newConcepts);
+  function getValidConcepts(allHierarchies) {
+    const newConcepts = Object.keys(allHierarchies).filter((key) => allHierarchies[key].includes(baseClass));
+    return newConcepts;
   }
 
-  function getEdgePredicates() {
-    return Object.entries(biolink.slots).map(([identifier, predicate]) => ({
-      predicate: strings.edgeFromBiolink(identifier),
-      domain: strings.nodeFromBiolink(predicate.domain),
-      range: strings.nodeFromBiolink(predicate.range),
-    }));
+  function dig(obj, keys, target) {
```

Comment:
Not sure what this function does from the name. Maybe a docstring?

Reply:
See 4df81a443890af3a6831529af6ae863c4583ca7b